WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -323,16 +323,38 @@ import WordPressFlux
         }
     }
 
+
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
         syncIfAppropriate()
 
+        debugCheckIfNeedToDisplaySelectInterests()
```

Comment:
Why this method was named this way?

Reply:
Because it's just debug code that will be removed once the presentation of the view is done in a later task. This PR is just creating the logic.